preprocessors/Emilia/config.json

```diff
@@ -0,0 +1,33 @@
+{
+    "language": {
+        "multilingual": true,
+        "supported": [
+            "zh",
+            "en",
+            "fr",
+            "ja",
+            "ko",
+            "de"
+        ]
+    },
+    "entrypoint": {
+        "input_folder_path": "examples",
+        "SAMPLE_RATE": 24000
+    },
+    "separate": {
+        "step1": {
+            "model_path": "/path/to/model/separate_model/UVR-MDX-NET-Inst_HQ_3.onnx",
```

Comment:
Do users need to modify the path? If so, add some comments. An example: [https://github.com/open-mmlab/Amphion/tree/main/egs/svc/MultipleContentsSVC#configuration](https://github.com/open-mmlab/Amphion/tree/main/egs/svc/MultipleContentsSVC#configuration)